utils/build-script-impl

```diff
@@ -154,10 +154,11 @@ KNOWN_SETTINGS=(
     enable-llvm-assertions      "1"              "set to enable llvm assertions"
     build-llvm                  "1"              "set to 1 to build LLVM and Clang"
     build-swift-tools           "1"              "set to 1 to build Swift host tools"
-    build-swift-stdlib          "1"              "set to 1 to build the Swift standard library"
-    build-swift-stdlib-unittest-extra "0"        "set to 1 to build optional StdlibUnittest components"
-    build-swift-sdk-overlay     "1"              "set to 1 to build the Swift SDK overlay"
+    build-swift-dynamic-stdlib  "1"              "set to 1 to build dynamic variants of the Swift standard library"
```

Comment:
Could we keep `build-swift-stdlib` and `build-swift-sdk-overlay` as aliases for `--build-swift-static-xyz=1 --build-swift-dynamic-xyz=1`?  We are using these in build presets, would be best to not break them.